mne/utils.py

```diff
@@ -2740,3 +2740,10 @@ def open_docs(kind=None, version=None):
         raise ValueError('version must be one of %s, got %s'
                          % (version, versions))
     webbrowser.open_new_tab('https://martinos.org/mne/%s/%s' % (version, kind))
+
+
+def _validate_type(item_name, item, type_name, types):
```

Comment:
This can be smarter:
```
..., types, type_name=None)

if type_name is None:
    iter_types = [types] if not isinstance(types, (list, tuple)) else types
    type_name = ', '.join(cls.__name__ for cls in iter_types)
```
(make the formatting as nice as you want, `', '.join(...)` is probably not sufficient)

Then maybe half of your calls don't need to provide `type_name`. If you add a special check for `string_types` then this jumps probably to 75%+ that don't need to.